### ISSUE TITLE
Clean install of the project will cause error

### ISSUE THREAD

**Issue by svenpopping** [WISVCH/events#79]
```
Exception in thread "main" java.lang.NoClassDefFoundError: org/springframework/boot/SpringApplication
	at ch.wisv.events.EventsApplication.main(EventsApplication.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
Caused by: java.lang.ClassNotFoundException: org.springframework.boot.SpringApplication
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 6 more

Process finished with exit code 1
```

**Comment by svenpopping** [comment]
It's a known issue in IntelliJ IDEA that is specific to Gradle 3.4

http://stackoverflow.com/questions/42587487/noclassdeffounderror-after-intellij-idea-upgrade/42588061#42588061

**Comment by praseodym** [comment]
It's fixed in 2017.1 anyways
